test/integration/webhooks/trainjob_test.go

```diff
@@ -150,7 +310,7 @@ var _ = ginkgo.Describe("TrainJob marker validations and defaulting", ginkgo.Ord
 			ginkgo.Entry("Should succeed to default runtimeRef.apiGroup",
 				func() *trainer.TrainJob {
 					return testingutil.MakeTrainJobWrapper(ns.Name, "empty-api-group").
-						RuntimeRef(schema.GroupVersionKind{Group: "", Version: "", Kind: trainer.TrainingRuntimeKind}, "testing").
+						RuntimeRef(trainer.SchemeGroupVersion.WithKind(trainer.TrainingRuntimeKind), "testing").
```

Comment:
```suggestion
						RuntimeRef(schema.GroupVersionKind{Group: "", Version: "", Kind: trainer.TrainingRuntimeKind}, "testing").
```

Is this change intended? IIUC, this case checks if defaulter can defaults in runtimeRef.

Reply:
Ah I see.